tests/integrations/import/sst_service.rs

```diff
@@ -137,10 +137,7 @@ fn test_download_sst() {
 
     let result = import.download(&download).unwrap();
     assert!(
-        result
-            .get_error()
-            .get_message()
-            .contains("CannotReadExternalStorage"),
```

Comment:
Why is it not `"CannotReadExternalStorage"`?

Reply:
Actually the error message is `Cannot read ...`, I'm confused why it can be passed for the first time.

Reply:
I found it returns
```
test import::sst_service::test_download_sst ... thread 'main' panicked at 'unexpected download reply: error { message: "Cannot read local:///tmp/test_download_sst9LnKJl/missing.sst" }', tests/integrations/import/sst_service.rs:139:5
```

Reply:
Seems the original error message is printed in `debug` mode and now is printed in `display` mode.😓